lua-aws/util.lua

```diff
@@ -36,10 +50,10 @@ _M.xml = (function ()
 				if not ni then break end
 				local text = string.sub(s, i, ni-1)
 				if not string.find(text, "^%s*$") then
-					if text:find('<?xml') then
+					if text:find('<?xml', 1, true) then
```

Comment:
Previously this was performing a regex pattern match.  That's not intended for this case.